app/src/main/java/org/gdg/frisbee/android/utils/Utils.java

```diff
@@ -209,8 +212,18 @@ public static boolean isEmailAddress(String possibleEmail) {
         return Patterns.EMAIL_ADDRESS.matcher(possibleEmail).matches();
     }
 
-
     public static boolean canLaunch(Context context, final Intent viewUrlIntent) {
         return context.getPackageManager().resolveActivity(viewUrlIntent, PackageManager.MATCH_DEFAULT_ONLY) != null;
     }
+
+    @SuppressWarnings("deprecation")
+    public static Intent createExternalIntent(Context context, Uri uri) {
+        Intent intent = new Intent(Intent.ACTION_VIEW, uri);
+        intent.putExtra(ShareCompat.EXTRA_CALLING_PACKAGE, context.getPackageName());

```

Comment:
This Extra's are taken from ShareCompat's implementation. The external Activity we open might use this information if they want to using ShareCompat.IntentReader.